pom.xml

```diff
@@ -401,6 +400,7 @@
         </plugin>
         <plugin>
           <artifactId>maven-dependency-plugin</artifactId>
+          <version>3.1.1</version>
```

Comment:
Created PHOENIX-7151, will take this separately.

Reply:
Please let me if you want me to do with this commit. Just don't want to complicate the patch, hence broke into another ticket.